### PR TITLE
DigiSub Discount confirmed v1

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
New screen which will be shown at the end of the DigiSub journey when a discount has been applied.

Also moved all of the existing storybooks about DigiSub Saves into one storybook file.

## How to test
No live urls so test via storybook

## Images
image

image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+	return (
+		<>
+			<section
+				css={css`
+					padding-top: ${space[4]}px;
+					margin-bottom: 32px;
+				`}
+			>
+				<h1 css={headingCss}>Discount confirmed</h1>
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx:76]: would this be the `h2` if "Your subscription" is the `h1`?

File: client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+						</li>
+						<li>
+							<SvgGift size="medium" />
+							<span>
+								<strong
+									css={css`
+										padding-bottom: ${space[1]}px;
+									`}
+								>
+									25% discount for 3 months
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx:119]: is it always 3 months 🤔

**Review by andrewHEguardian — APPROVED** [review]
Looks good! 🚀

File: client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx (hunk 2 above)
Comment (charleycampbell) [reply client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx:119]: I thought so...This screen still needs data inputted so I'm going to leave it and we can ask Daniel when it comes to inputting the other values

Comment (charleycampbell) [reply client/components/mma/cancel/cancellationSaves/digipack/DigiSubDiscountConfirm.tsx]: Daniel is going to confirm so I have popped in a placeholder

**Review by andrewHEguardian — APPROVED** [review]

**Pull request merged**